src/components/ui/Facets/Selections.tsx

```diff
@@ -61,6 +61,8 @@ function Selections() {
 			return `${startDate} - ${endDate}`;
 		} else if (selection.facet.FieldType === 'range') {
 			return renderRange(item, selection.facet);
+		} else if (selection.facet.FacetType === 'search') {
+			return decodeURIComponent(item.label);
```

Comment:
I don't think this is required - label comes from the response, not from the url

Reply:
it is extracting it from pending search